lib/template/config/initializers/rollbar.rb

```diff
@@ -1,6 +1,7 @@
 Rollbar.configure do |config|
   config.enabled = ENV.has_key?('ROLLBAR_ACCESS_TOKEN')
   config.access_token = ENV["ROLLBAR_ACCESS_TOKEN"]
+  config.environment = ENV['ROLLBAR_ENVIRONMENT']
```

Comment:
Would it make sense to add `ROLLBAR_ENVIRONMENT` to [`pliny/lib/template/.env.sample.erb`](https://github.com/interagent/pliny/blob/fdf30c530367d07bae2064527019bfd38edbf7dc/lib/template/.env.sample.erb)?

```
ROLLBAR_ENVIRONMENT=development
```